app/models/sponsor.rb

```diff
@@ -5,6 +5,7 @@ class Sponsor < ActiveRecord::Base
   PRIORITY_COUNTRIES= %w(SA TR AE GB)
   EXCLUDED_COUNTRYS= %w(IL)
 
+  self.per_page = 10
```

Comment:
As in my comment on #310 - can we find a different place to store these settings?